data/MCScript/submissions/trian/trian_dev_submission.jsonl

```diff
@@ -0,0 +1,1411 @@
+{"sample_id": 0, "question_id": 0, "pred": 1}
```

Comment:
As we discussed, I'd prefer not to have redundant intermediate formats. Read the .txt file in the transformer (or the extractor, if you want to do it that way), don't create a redundant translation.